perun/fuzz/evaluate/by_perun.py

```diff
@@ -31,9 +31,11 @@ def baseline_testing(
     postprocessor: list[str],
     minor_version_list: list[MinorVersion],
     **kwargs: Any,
-) -> Iterable[tuple[CollectStatus, Profile, str]]:
+) -> Iterable[tuple[CollectStatus, Profile, Job]]:
     """Generates a profile for specified command with init seeds, compares each other.
 
+    TODO: This might need some checking and tweaking as I believe it is quite shady
```

Comment:
What's shady about this? Is it something serious? If so, we should probably open a new issue for it so we don't forget about it.